cmd/cluster/powervs/create.go

```diff
@@ -188,13 +188,13 @@ func (o *CreateOptions) ApplyPlatformSpecifics(cluster *hyperv1.HostedCluster) e
 			Region:            o.Region,
 			Zone:              o.Zone,
 			CISInstanceCRN:    o.infra.CISCRN,
-			ServiceInstanceID: o.CloudInstanceID,
+			ServiceInstanceID: o.infra.CloudInstanceID,
 			Subnet: &hyperv1.PowerVSResourceReference{
 				Name: &o.infra.DHCPSubnet,
 				ID:   &o.infra.DHCPSubnetID,
 			},
 			VPC: &hyperv1.PowerVSVPC{
-				Name:   o.VPC,
+				Name:   o.infra.VPCName,
```

Comment:
It is similar to the usage of Region [here](https://github.com/openshift/hypershift/blob/c835c34db51a2c0fa474ecd67beb4cfb78e17db4/cmd/cluster/aws/create.go#L227), which is user passed and not going to get changed. Whereas VPC is created during infra setup process and since we are appending a suffix to it during infra setup process, need to fetch it from infra. Also `o.VPC` is not meant to pass the VPC name to create, it's meant to pass existing resource.